HW4/task2.py

```diff
@@ -0,0 +1,19 @@
+# Напишите функцию, принимающую на вход только ключевые параметры и возвращающую словарь,
+# где ключ — значение переданного аргумента, а значение — имя аргумента.
+# Если ключ не хэшируем, используйте его строковое представление.
+
+def create_dictionary(**kwargs):
+    result = {}
+    for key, value in kwargs.items():
+        if not isinstance(value, (int, float, str)):
+            value = repr(value)
+        if not isinstance(key, (int, float, str)):
+            key = str(key)
```

Comment:
Вы проверяете основные, но далеко не все типы данных. Значение может быть любым
```suggestion
        if not isinstance(value, (int, float, str)):
            value = str(value)
```